evm/loomevm.go

```diff
@@ -54,44 +49,24 @@ type LoomEvm struct {
 // TODO: this doesn't need to be exported, rename to newLoomEvmWithState
 func NewLoomEvm(
 	loomState loomchain.State, accountBalanceManager AccountBalanceManager,
-	logContext *ethdbLogContext, debug bool,
+	logContext *store.EthDBLogContext, debug bool,
 ) (*LoomEvm, error) {
-	p := new(LoomEvm)
-	p.db = NewLoomEthdb(loomState, logContext)
-	oldRoot, err := p.db.Get(rootKey)
-	if err != nil {
-		return nil, err
-	}
-
+	p := &LoomEvm{}
 	var abm *evmAccountBalanceManager
+	var err error
 	if accountBalanceManager != nil {
 		abm = newEVMAccountBalanceManager(accountBalanceManager, loomState.Block().ChainID)
-		p.sdb, err = newLoomStateDB(abm, common.BytesToHash(oldRoot), state.NewDatabase(p.db))
+		p.sdb, err = newLoomStateDB(abm, loomState.EVMState().StateDB())
```

Comment:
`LoomEvm` no longer reads or writes from the `MultiWriterAppStore` because the `StateDB` here no longer wraps `loomchain.State`, so we can eliminate all the reads & writes of `EvmStore` from `MultiWriterAppStore`, right?

Reply:
yes